src/main/java/org/hiero/sketch/table/SortOrder.java

```diff
@@ -0,0 +1,47 @@
+package org.hiero.sketch.table;
+
+import org.hiero.sketch.table.api.IRowIterator;
+import org.hiero.sketch.table.api.IRowOrder;
+import org.hiero.sketch.table.api.IndexComparator;
+
+import java.util.Arrays;
+
+/**
+ * Given a (table and a) IndexComparator, gives an iterator for the rows of the table in sorted order
```

Comment:
I would call this ExplicitSortOrder.
You should use a comment that this is expected to be small.